U-Boot can't yet parse / apply overlays referencing local fragment aliases.

diff --git a/arch/arm/boot/dts/overlays/cma-overlay.dts b/arch/arm/boot/dts/overlays/cma-overlay.dts
index 1d87c599f909d..bb1394e34f5c5 100644
--- a/arch/arm/boot/dts/overlays/cma-overlay.dts
+++ b/arch/arm/boot/dts/overlays/cma-overlay.dts
@@ -10,7 +10,7 @@
 
 	fragment@0 {
 		target = <&cma>;
-		frag0: __overlay__ {
+		__overlay__ {
 			/*
 			 * The default size when using this overlay is 256 MB
 			 * and should be kept as is for backwards
@@ -21,16 +21,16 @@
 	};
 
 	__overrides__ {
-		cma-512 = <&frag0>,"size:0=",<0x20000000>;
-		cma-448 = <&frag0>,"size:0=",<0x1c000000>;
-		cma-384 = <&frag0>,"size:0=",<0x18000000>;
-		cma-320 = <&frag0>,"size:0=",<0x14000000>;
-		cma-256 = <&frag0>,"size:0=",<0x10000000>;
-		cma-192 = <&frag0>,"size:0=",<0xC000000>;
-		cma-128 = <&frag0>,"size:0=",<0x8000000>;
-		cma-96  = <&frag0>,"size:0=",<0x6000000>;
-		cma-64  = <&frag0>,"size:0=",<0x4000000>;
-		cma-size = <&frag0>,"size:0"; /* in bytes, 4MB aligned */
+		cma-512 = <0>,"size:0=",<0x20000000>;
+		cma-448 = <0>,"size:0=",<0x1c000000>;
+		cma-384 = <0>,"size:0=",<0x18000000>;
+		cma-320 = <0>,"size:0=",<0x14000000>;
+		cma-256 = <0>,"size:0=",<0x10000000>;
+		cma-192 = <0>,"size:0=",<0xC000000>;
+		cma-128 = <0>,"size:0=",<0x8000000>;
+		cma-96  = <0>,"size:0=",<0x6000000>;
+		cma-64  = <0>,"size:0=",<0x4000000>;
+		cma-size = <0>,"size:0"; /* in bytes, 4MB aligned */
 		cma-default = <0>,"-0";
 	};
 };
